src/Reflection/ReflectionClassConstant.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace Roave\BetterReflection\Reflection;
+
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\ClassConst;
+use Roave\BetterReflection\NodeCompiler\CompileNodeToValue;
+use Roave\BetterReflection\NodeCompiler\CompilerContext;
+use Roave\BetterReflection\Reflector\Reflector;
+
+class ReflectionClassConstant implements \Reflector
+{
+
+    /**
+     * @var bool value of const was cached?
+     */
+    private $valueCached = false;
+
+    /**
+     * @var mixed const value
```

Comment:
Constants are not mixed: they can contain `int|float|array|string|bool|null` - all related docblocks need alignment to this too